### PR TITLE
[DP-420] add autocontinue flag for rebalance

### PR DESCRIPTION
**Pull request by hhahn-tw**
Provides `--auto-continue-rebalance` option that will skip prompting after each rebalance round (this is separate from the prompt to start the rebalance, which can be skipped via `--skip-confirm`).

Also added some output to indicate progress via "Balancing round X of Y" output during rebalance.

### CODE REVIEW

File: pkg/apply/apply.go [hunk 1]
```diff
@@ -835,6 +837,10 @@ func (t *TopicApplier) updatePlacementRunner(
 		return nil
 	}
 
+	if t.config.AutoContinueRebalance {
+		log.Infof("Autocontinue flag detected, user will not be prompted each round")
```
Comment (petedannemann) [inline pkg/apply/apply.go:841]: Nit but I might use `log.Warnf` here

File: pkg/apply/apply.go [hunk 2]
```diff
@@ -854,13 +860,22 @@ func (t *TopicApplier) updatePlacementRunner(
 		)
 	}
 
+	numRounds := (len(assignmentsToUpdate) + batchSize - 1) / batchSize // Ceil() with integer math
+	round := 0
```
Comment (petedannemann) [inline pkg/apply/apply.go:864]: What do you think about refactoring this for loop to use the round and numRounds variables? It is a bit confusing how we are tracking the iteration count in two ways right now

File: pkg/apply/apply.go (hunk 1 above)
Comment (hhahn-tw) [reply pkg/apply/apply.go:841]: Good idea

File: pkg/apply/apply.go (hunk 2 above)
Comment (hhahn-tw) [reply pkg/apply/apply.go:864]: I didn't really like having two separate counters but I tried a couple of variations and none seem any better. If I use `round` as the counter, I have to calculate `i` from it (eg `i := round * batchSize`) anyway.

If I keep the loop based on `i`, but calculate `rounds` based on `i` (`round := i/batchSize+1`), it doesn't seem than just independently incrementing it.

Any other ideas?

Comment (petedannemann) [reply pkg/apply/apply.go:865]: Ah, yeah maybe I am off base here. Does initializing both iterables and incrementing them in the for loop initialize like this seem any cleaner?

```go
for i, round := 0, 0; i < len(assignmentsToUpdate); i += batchSize, round += 1
```

Comment (hhahn-tw) [reply pkg/apply/apply.go]: Hmm, I've never tried that before. It's unusual and a little harder to read but I like that it explicitly pins the round counter to the `for` statement 👍

**Review by petedannemann — APPROVED** [review]

File: pkg/version/version.go [hunk 3]
```diff
@@ -1,4 +1,4 @@
 package version
 
 // Version is the current topicctl version.
-const Version = "1.5.0"
```
[the remark this answers is not in the document]
Comment (hhahn-tw) [reply pkg/version/version.go:4]: Updated

**Pull request merged**